### PR TITLE
Minor fable-library update

### PR DESCRIPTION
**Pull request by ncave**

### CODE REVIEW

File: tests/Main/TypeTests.fs [hunk 1]
```diff
@@ -260,22 +260,22 @@ type TypeWithClassAttribute =
 
 // -------------------------------------------------------------
 // Issue #1975: https://github.com/fable-compiler/Fable/issues/1975
-// In previous version of Fable, using type with parametrized units of measure was causing an endless loops in the compiler
+// In previous version of Fable, using type with parameterized units of measure was causing an endless loops in the compiler
```
Comment (MangelMaxime) [inline tests/Main/TypeTests.fs]: Thank you for catching my typos :)

**Comment by MangelMaxime** [comment]
@ncave I reverted the change to `global.json` because otherwise it's blocking me on Gitpod and I need to use it now otherwise I can't really work on Fable etc. (my workstation has changed at work)

I suspect gitpod or ubuntu or docker don't have yet the latest version of dotnet core which was released yesterday.

**Comment by ncave** [comment]
Sure, sorry about that, I should have tried in it gitpod first.
Btw, do F# extensions work for you in Gitpod/Theia?

**Comment by MangelMaxime** [comment]
> Sure, sorry about that, I should have tried in it gitpod first.

No problem :)

> Btw, do F# extensions work for you in Gitpod/Theia?

Yes it does (sill a beta feature). 

```yaml
vscode:
  extensions:
    - christian-kohler.path-intellisense@1.4.2:QnOrf5fk6KiVaQs4cNEP+w==
    - wayou.vscode-todo-highlight@1.0.4:8IqxuxCVol2WnScJc5xVzg==
    - mrmlnc.vscode-scss@0.9.0:/wXbNRm+2kunH5HbQqfnXA==
    - PKief.material-icon-theme@3.9.2:xeHlNzPEF04yFqz/xKCD5w==
    - Ionide.Ionide-fsharp@4.5.0:0qxXuhq6eO066etkNQrKCQ==
    - zhuangtongfa.material-theme@3.2.2:jGTZwg0ChZg3eEKHC+UO+w==
    - mhutchie.git-graph@1.21.0:zoSeoEOMfrwN0BMBC8VvqQ==
    - eamodio.gitlens@10.2.0:GNDO73Cmp0fYDiLNxVkbsQ==
```

The extensions are configured in the [`gitpod.yml` file](https://github.com/fable-compiler/Fable/blob/master/.gitpod.yml#L28-L37). I should clean it a bit because `git-graph` is not working and I should probably not impose the theme used etc.

In Gitpod, you can have extensions installed per repo via the `gitpod.yml` file or per user . The later will be loaded for you each time you open a repo via gitpod. So if you like a theme, you can have it always installed etc.

**Comment by ncave** [comment]
@MangelMaxime Somehow ionide wasn't working for me (on win10) on gitpod, but I haven't tried enough variation of browsers/os yet. Otherwise your setup looks good.

**Comment by MangelMaxime** [comment]
@ncave 

Let me check because it seems like on the screenshot I have Ionide installed as an extension for me (as a user). 

But other than that I am on win10 and I am using google chrome. For now, using Firefox is not recommended because there is a bug in Monaco editor which doesn't handle correctly the scrollbar.

**Comment by MangelMaxime** [comment]
@ncave Ok I forced a `dotnet restore` during the initalisation of the workspace and fix a case typo in the sln file. It should work out of the box now.

**Pull request merged**